packages/html/src/slider/templates/slider-vertical.tsx

```diff
@@ -0,0 +1,20 @@
+import { Slider, SliderTick } from "../../slider";
+
+export const SliderVertical = (props) => (
+    <Slider orientation="vertical" children={[
```

Comment:
We do not need to pass the **children** as an array here - `children={ <> JSX.Elements </> }` should be enough.
It applies to the other templates as well.
```suggestion
    <Slider orientation="vertical" children={
```